pkg/landscape/app/graph-store.hoon

```diff
@@ -111,7 +112,7 @@
   ?>  (team:title our.bowl src.bowl)
   =^  cards  state
     ?+  mark           (on-poke:def mark vase)
-      %graph-update-2  (graph-update !<(update:store vase))
+      %graph-update-3  (graph-update !<(update:store vase))
```

Comment:
Are these not backwards compatible in some way? What changed?

Reply:
references to other apps were added

Reply:
We can and should continue accepting `%graph-update-2` then, right?